example_regression_2.py

```diff
@@ -0,0 +1,114 @@
+"""Load a saved regression model, make predictions of various types and save them to a csv or database.
+
+Please use this example to learn about healthcareai before moving on to the next example.
+
+If you have not installed healthcare.ai, refer to the instructions here:
+  http://healthcareai-py.readthedocs.io
+
+To run this example:
+  python3 example_regression_2.py
+
+This code uses the DiabetesClinicalSampleData.csv source file.
+"""
+import pandas as pd
+
+import healthcareai.common.file_io_utilities as hcai_io_utilities
+import healthcareai.common.database_connections as hcai_db
+
+
+def main():
+    # Load data from a sample .csv file
+    prediction_dataframe = pd.read_csv('healthcareai/tests/fixtures/DiabetesClinicalSampleData.csv', na_values=['None'])
+
+    # Load data from a MSSQL server: Uncomment to pull data from MSSQL server
+    # server = 'localhost'
+    # database = 'SAM'
+    # query = """SELECT *
+    #             FROM [SAM].[dbo].[DiabetesClincialSampleData]
+    #             WHERE SystolicBPNBR is null"""
+    #
+    # engine = hcai_db.build_mssql_engine(server=server, database=database)
+    # prediction_dataframe = pd.read_sql(query, engine)
+
+    # Drop columns that won't help machine learning
+    columns_to_remove = ['PatientID']
+    prediction_dataframe.drop(columns_to_remove, axis=1, inplace=True)
+
+    # Load the saved model using the correct file name (they look like
+    #     '2017-05-31T15-55-55_regression_LinearRegression.pkl')
```

Comment:
Perhaps say something like, "we are using the model we created in regression example 1." ??